packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2734,7 +2734,7 @@ export class StripeHelper extends StripeHelperBase {
     };
   }
 
-  stripePlanToPaymentCycle(plan: Stripe.Plan) {
+  async stripePlanToPaymentCycle(plan: Stripe.Plan) {
```

Comment:
This isn't an asynchronous method. `async` isn't necessary here.

Reply:
Good catch. I was trying something and meant to remove this when I was done. Removed!